lib/browser/mqtt.ts

```diff
@@ -576,7 +577,8 @@ export class MqttClientConnection extends BufferedEventEmitter {
                     return this.on_error(error);
                 }
                 const sub = (packet as mqtt.ISubscriptionGrant[])[0];
-                resolve({ topic: sub.topic, qos: sub.qos });
+                // MV TOFIX: 128 can be passed here and is not modeled in QoS
+                resolve({ topic: sub.topic, qos: sub.qos as QoS });
```

Comment:
Ideally, it should not be an issue as we would throw error before we reach here when returned qos is 128.